data/CREDITS

```diff
@@ -91,7 +90,8 @@ Bug fixes & misc contributions
 - Stephen Dunn (Work on audio)
 - Patrick Ammann
 - Ward Muylaert
-
+- Fantasmos
+- QwertyChouskie
```

Comment:
Qwerty is a frequent IRC follower but I don't think he's contributed much code - no ill intentions, but for practical reasons we can't list all minor contributors in the credits IMO, otherwise they would be much much larger.

Reply:
I based it on the data found here : https://github.com/supertuxkart/stk-code/commits?author=qwertychouskie

The contributions listed there seem to fit in "misc contributions" for me ; and many small improvements (donate/reset password buttons ; new skidding particles ; several small bugfixes and build improvements) end up significant.

Considering the credits list some people for things which aren't in the game anymore (e.g., removed kart) or sometimes even for one particular improvement, I think he has a better case to be in there than many others who already are.

This is free software, we're volunteers, so I don't think it's too much to repay time and effort with an acknowledgement.

Oh, and the credits list donators, too. Listing even all minor contributors wouldn't even approach the size of the donation list.